### PR TITLE
BAU Set gateway account corporate surcharge feature

### PR DESCRIPTION
**Pull request by sfount**
* update connector request utility to update surcharge values from connector
* link together gateway accounts and update surcharge utils

Screen Shot 2019-08-06 at 14 16 21

### CODE REVIEW

**Comment by oswaldquek** [comment]
one for @jonheslop to review

File: src/web/modules/gateway_accounts/surcharge.njk [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  <div>
+    <a href="/gateway_accounts/{{ account.gateway_account_id }}" class="govuk-back-link">Gateway account ({{ account.gateway_account_id }})</a>
+  </div>
+
+  <form method="POST" action="/gateway_accounts/{{ account.gateway_account_id }}/surcharge">
+    {{ govukInput({
+      id: "corporate_credit_card_surcharge_amount",
+      name: "corporate_credit_card_surcharge_amount",
+      value: account.corporate_credit_card_surcharge_amount or "0",
+      label: { text: "Corporate credit card surcharge amount" }
```
Comment (alexbishop1) [inline src/web/modules/gateway_accounts/surcharge.njk]: Could you make this “Corporate non-prepaid credit card surcharge amount” for clarity?

File: src/web/modules/gateway_accounts/surcharge.njk [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      })
+    }}
+
+    {{ govukInput({
+      id: "corporate_debit_card_surcharge_amount",
+      name: "corporate_debit_card_surcharge_amount",
+      value: account.corporate_debit_card_surcharge_amount or "0",
+      label: { text: "Corporate debit card surcharge amount" }
```
Comment (alexbishop1) [inline src/web/modules/gateway_accounts/surcharge.njk]: Could you make this “Corporate non-prepaid debit card surcharge amount” for clarity?

**Comment by sfount** [comment]
@alexbishop1 https://github.com/alphagov/pay-toolbox/pull/180/commits/d662b5931d1152b78830dafa024df4e8d09011a0

**Pull request merged**